cmd/generate/generate.go

```diff
@@ -159,15 +159,15 @@ func Validate(o *GenerateOptions) error {
 			o.format, strings.Join(validFormats, ", "))
 	}
 
-	if o.customEntriesPath == "" {
-		return nil
+	if o.customEntriesPath != "" && o.customEntriesFormat == "" {
+		return fmt.Errorf("you must specify the --customEntriesFormat when using --customEntriesPath")
 	}
 
-	if o.customEntriesFormat == "" {
-		return fmt.Errorf("you must specify the --customEntriesFormat when using --customEntriesPath")
+	if o.customEntriesPath == "" && o.customEntriesFormat != "" {
+		return fmt.Errorf("you must specify the --customEntriesPath when using --customEntriesFormat")
 	}
 
-	if !slices.Contains(validCustomEntriesFormats, o.customEntriesFormat) {
+	if o.customEntriesFormat != "" && !slices.Contains(validCustomEntriesFormats, o.customEntriesFormat) {
```

Comment:
when you return nil on both empty, you won't need the `o.customEntriesFormat != ""`